domain/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
see https://github.com/emotz/kladovka/issues/97 for reference, no need to change in this pull request